man1/aur-vercmp.1

```diff
@@ -29,6 +29,12 @@ versions in the AUR, and print updates to stdout.
 The name of a pacman repository. If \fInot\fR specified, packages and their
 versions are taken from stdin (tab-separated).
 
+.TP
+.B \-p
+Read package versions from a file instead of the AUR. The package list should
+be in the same format as stdin (\fIpkgname\fR as the first
+field, \fIpkgver\fR as the second, both separated by tabs).
```

Comment:
Post https://github.com/AladW/aurutils/commit/fe18afaaaeb596d103f76d503e4ad082637c1f0a any blanks (not just tabs) should do. Looks like I forgot to update the paragraph above